src/compiler/types.ts

```diff
@@ -2547,6 +2547,7 @@ namespace ts {
         Warning,
         Error,
         Message,
+        CodeFix,

```

Comment:
AFAIK this seems no different than `Message`. We should probably just reuse `Message`.